pkgs/modules/default.nix

```diff
@@ -10,7 +10,6 @@ let
       python = pkgs.python310Full;
       pypkgs = pkgs.python310Packages;
     }))
-    (mkModule ./pyright-extended)
```

Comment:
I was thinking yes. We don't really have another good way to distribute it.

Reply:
Oh I see, it would have the normal python config in .replit plus the pyright-extended module.

Reply:
Yes, with the normal LSP config removed.

Reply:
Thinking about 2 options:

1. re-use a smaller chunk of pyright-extended rather than a whole module
2. keep the lsp in a separate module: even do this in general. Add both modules in .replit. I've been considering doing this for the typescript-language-server too, since this would eliminate the problem of running multiple instances of the same lsp.

Reply:
3rd option: just go straight for rolling out Python + pyright.

Reply:
I think 1 is the faster/simpler option for now. 2. means we have to make upgrade maps for each one and reason about that.

Reply:
:+1: